csharp/ql/test/library-tests/enums/PrintAst.expected

```diff
@@ -11,18 +11,35 @@ enums.cs:
 #   23|   3: [Enum] E
 #   25|   4: [Enum] ValueColor
 #   28|     5: [Field] OneRed
-#   28|       1: [MemberConstantAccess] access to constant OneRed
+#   28|       1: [AssignExpr] ... = ...
+#   28|         0: [CastExpr] (...) ...
+#   28|           0: [IntLiteral] 1
+#   28|         1: [MemberConstantAccess] access to constant OneRed
 #   29|     6: [Field] TwoGreen
-#   29|       1: [MemberConstantAccess] access to constant TwoGreen
+#   29|       1: [AssignExpr] ... = ...
+#   29|         0: [CastExpr] (...) ...
+#   29|           0: [IntLiteral] 2
+#   29|         1: [MemberConstantAccess] access to constant TwoGreen
 #   30|     7: [Field] FourBlue
-#   30|       1: [MemberConstantAccess] access to constant FourBlue
+#   30|       1: [AssignExpr] ... = ...
+#   30|         0: [CastExpr] (...) ...
+#   30|           0: [IntLiteral] 4
+#   30|         1: [MemberConstantAccess] access to constant FourBlue
 #   34|   5: [Enum] SparseColor
 #   37|     5: [Field] Red
 #   38|     6: [Field] Green
-#   38|       1: [MemberConstantAccess] access to constant Green
+#   38|       1: [AssignExpr] ... = ...
+#   38|         0: [IntLiteral] 10
+#   38|         1: [MemberConstantAccess] access to constant Green
 #   39|     7: [Field] Blue
 #   40|     8: [Field] AnotherBlue
-#   40|       1: [MemberConstantAccess] access to constant AnotherBlue
+#   40|       1: [AssignExpr] ... = ...
+#   40|         0: [AddExpr] ... + ...
+#   40|           0: [CastExpr] (...) ...
```

Comment:
@hvitved These casts look a bit strange. How come we have no `TypeAccess` below these nodes?